### ISSUE TITLE
RequesterUtils.d.ts:28:39 - error TS2304: Cannot find name 'Headers'.

### ISSUE THREAD

**Issue by whyour** [jdalrymple/gitbeaker#2295]
```bash
node_modules/@gitbeaker/requester-utils/dist/types/RequesterUtils.d.ts:28:39 - error TS2304: Cannot find name 'Headers'.

28     headers: Record<string, string> | Headers;
```
**Steps to reproduce**

tsc build

**Expected behaviour**

no error

**Actual behaviour**

throw error

**Possible fixes**

delete Headers type

**Comment by patrick-webs** [comment]
I was able to get past this error by changing my tsconfig.json `compilerOptions` from

    "lib": ["ES2021"],

to

    "lib": ["ES2021", "DOM"],

**Comment by jdalrymple** [comment]
Ill see if i can get this sorted on this end. Possibly doing the same change.

**Comment by jdalrymple** [comment]
Its set this way in the codebase, im not sure there is much else i can do!

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`37.0.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/37.0.0) :rocket:
